pkgs/top-level/linux-kernels.nix

```diff
@@ -541,6 +541,8 @@ in {
 
     turbostat = callPackage ../os-specific/linux/turbostat { };
 
+    corefreq = callPackage ../os-specific/linux/corefreq { };
```

Comment:
Strange, can't it be sent to by-name?

Reply:
Is there a `by-name` pattern for kernel modules? It needs to be part of the kernelPackages scope in order to be injected the right `kernel` dependency.

Reply:
Hum, so it can't be sent yet. Let's leave it as is.